### PR TITLE
Stage AIO env - Adding missing env variables for Stage AIO environment during deploy to stage

### PR DESCRIPTION
**Pull request by arshadparwaiz**

### CODE REVIEW

File: .github/workflows/deploy_stage.yml [hunk 1]
```diff
@@ -38,12 +38,15 @@ jobs:
...
           BULK_PREVIEW_CHECK_INTERVAL: ${{ vars.BULK_PREVIEW_CHECK_INTERVAL }}
-          MAX_BULK_PREVIEW_CHECKS: ${{ vars.MAX_BULK_PREVIEW_CHECKS }}
+          CERT_THUMB_PRINT: ${{ secrets.CERT_THUMB_PRINT }}
+          CLIENT_ID: ${{ secrets.CLIENT_ID }}
+          ENABLE_PREVEW: ${{ vars.ENABLE_PREVEW }}
           GRAYBOX_USER_GROUPS: ${{ vars.GRAYBOX_USER_GROUPS }}
+          GROUP_CHECK_URL: ${{ vars.GROUP_CHECK_URL }}
+          MAX_BULK_PREVIEW_CHECKS: ${{ vars.MAX_BULK_PREVIEW_CHECKS }}
           SPLUNK_HEC__HEC_TOKEN: ${{ secrets.SPLUNK_HEC_TOKEN_STAGE }}
+          TENANT_ID: ${{ secrets.TENANT_ID }}
```
Comment (sukamat) [inline .github/workflows/deploy_stage.yml:49]: @arshadparwaiz It is still missing a few I think. 
`app.config.yaml` file reads these values and maps to the variables.
https://github.com/adobecom/graybox-io/blob/main/app.config.yaml#L8-L21

**Review by sukamat — CHANGES_REQUESTED** [review]

File: .github/workflows/deploy_stage.yml (hunk 1 above)
Comment (sukamat) [reply .github/workflows/deploy_stage.yml:52]: Those should be added then. I am assuming your .env file already has those values.

Comment (arshadparwaiz) [reply .github/workflows/deploy_stage.yml:55]: Added the secrets in github and then updated the vars in deploy_stage.yaml

Comment (arshadparwaiz) [reply .github/workflows/deploy_stage.yml:55]: Thanks Sunil

**Review by sukamat — APPROVED** [review]

**Pull request merged**